nic_api/__init__.py

```diff
@@ -18,7 +18,7 @@
 
 DEFAULT_TTL = 600
 
-_RECORD_CLASSES = (ARecord, CNAMERecord)
+_RECORD_CLASSES = (ARecord, CNAMERecord, TXTRecord, SOARecord, NSRecord, MXRecord))
```

Comment:
There's extra bracket in the end. Also I'm not sure that it's possible to add SOA records by API, I will check it.

Reply:
Sorry,  bracket - my mistake. I fix it. 
SOA is just for read. I will need read all records of domain and out theirs names for my project.

Reply:
`_RECORD_CLASSES` is used for `DnsApi.add_record()` method to check if records passed to method can be created in zone. I think it's impossible to create more than 1 SOA in zone.

I will rename this variable for better code readability.